react/src/tracker.js

```diff
@@ -1,23 +1,53 @@
-import { newTracker, trackPageView, enableActivityTracking } from "@snowplow/browser-tracker";
-import React from 'react';
-import { useLocation } from 'react-router-dom';
-import { FormTrackingPlugin } from '@snowplow/browser-plugin-form-tracking';
-import { YouTubeTrackingPlugin } from '@snowplow/browser-plugin-youtube-tracking';
+import {
+  newTracker,
+  trackPageView,
+  enableActivityTracking,
+} from "@snowplow/browser-tracker";
+import React from "react";
+import { useLocation } from "react-router-dom";
+import { FormTrackingPlugin } from "@snowplow/browser-plugin-form-tracking";
+import { YouTubeTrackingPlugin } from "@snowplow/browser-plugin-youtube-tracking";
+import { SnowplowMediaPlugin } from "@snowplow/browser-plugin-media";
 
-let tracker = newTracker('ns1', 'http://localhost:9090', {
-  plugins: [FormTrackingPlugin(), YouTubeTrackingPlugin()]
-});
+var tracker;
 
-enableActivityTracking({
-  minimumVisitLength: 5,
-  heartbeatDelay: 5
-});
+const initializeTracker = (endpoint) => {
+  tracker = newTracker("ns1", "http://localhost:9090", {
+    plugins: [
+      FormTrackingPlugin(),
+      YouTubeTrackingPlugin(),
+      SnowplowMediaPlugin(),
+      {
```

Comment:
Can we maybe create this custom plugin in another file and give it a name and a small description ?

Reply:
Good point! Have moved the plugin to a separate file.